packages/clients/meldemichel/src/utils/createMenus.ts

```diff
@@ -12,6 +14,28 @@ export default function (mode: keyof typeof MODE): Menu[] {
       icon: 'fa-book-atlas',
       id: 'layerChooser',
     },
+    {
+      plugin: Gfi({
+        displayComponent: true,
+        gfiContentComponent: MeldemichelGfiFeature,
+        coordinateSources: [
+          'plugin/pins/transformedCoordinate',
+          'plugin/pins/coordinatesAfterDrag',
```

Comment:
In DISH I stumbled over `coordinatesFromMarker: 'plugin/pins/coordinatesAfterDrag',`, which is not documented or anywhere in the code.  
Do you know if this has been deleted by accident or was never present?

Reply:
It does exist in the Pins plugin. I assume we could remove it with some refactoring, but from my memory there were edge cases where it was the only state variable holding the pin's current coordinate. Looking at the name, it was probably after dragging.

Reply:
![where](https://github.com/Dataport/polar/assets/73653210/045e035d-1ce6-4b8b-b2f0-8b783a3a685d)

I can't seem to find where `coordinatesFromMarker` is used. Can you point me to the location?

Reply:
Oh, this isn't about `plugin/pins/coordinatesAfterDrag`. Yeah, I don't know what `coordinatesFromMarker` is, either.

Reply:
I've globally removed it and checked affected clients; the behaviour did not change (as expected). https://github.com/Dataport/polar/pull/30/commits/78344eb81bebe7b2ef587dc65b54cac8ec363138